helm_deploy/values-dev.yaml

```diff
@@ -19,13 +18,13 @@ generic-service:
     BANNER_EMPTY:
     BANNER_DPSMENU:
 
-  allowlist:
-    hpa-iis-stage-1: "51.141.36.127/32"
-    hpa-iis-stage-2: "51.141.34.169/32"
-    hpa-iis-stage-3: "51.141.33.76/32"
-    hpa-iis-stage-4: "51.141.38.101/32"
-    hpa-iis-stage-5: "51.140.210.98/32"
 
+  allowlist:
+    hpa-iis-preprod-1: 51.141.36.127/32
```

Comment:
We perhaps want to rename these back to 'stage'? Looks like stage and preprod IPs are the same though, maybe @ravmoj can advise what's best to call them.